### PR TITLE
Tailwind gray switcheroo

### PR DESCRIPTION
**Pull request by weerd**
### What's this PR do?

This pull request continues the Tailwind updates and in particular switches the currently neutral gray palette to the new cool gray palette (default Tailwind grays) on Phoenix. Along with changes to the new darker yellow.

### How should this be reviewed?

👀 

### Any background context you want to provide?

Example design with old neutral grays:
Old Neutral Grays

Example design with new cool grays:
New Cool Grays

### Relevant tickets

References [Pivotal #169578779](https://www.pivotaltracker.com/story/show/169578779).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [x] Added screenshots of front-end changes on small, medium, and large screens.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -106,6 +96,20 @@ module.exports = {
...
+        '70': 'rgba(0,0,0,0.7)',
+        '90': 'rgba(0,0,0,0.9)',
+      },
+      'transparent-white': {
+        '10': 'rgba(255,255,255,0.1)',
+        '30': 'rgba(255,255,255,0.3)',
+        '50': 'rgba(255,255,255,0.5)',
+        '70': 'rgba(255,255,255,0.7)',
+        '90': 'rgba(255,255,255,0.9)',
+      },
```
Comment (weerd) [inline tailwind.config.js]: While we want to refrain from using transparent black and white, we needed access to some values. An example that popped up during this work related to the border on the `Card` component. Depending on the background the card appeared on, an opaque gray color for the border looked on with some background colors, so it made sense to allow the borders to be a transparent black and thus absorb a bit of the background color in them and blend better in the context.

File: resources/assets/components/Revealer/revealer.scss [hunk 2]
```diff
@@ -1,5 +1,7 @@
 @import '../../scss/next-toolbox';
 
+$light-background-color: #f7fafc; // theme('colors.gray.100')
```
Comment (DFurnes) [inline resources/assets/components/Revealer/revealer.scss]: Is this comment meant to replace the Sass variable? Should we add a `TODO` explaining it?

Comment (weerd) [reply resources/assets/components/Revealer/revealer.scss]: I can add a TODO. The problem is that using the `theme()` function inside `rgba()` throws an error, so ended up leaving this as the variable and using the hex version of the gray Tailwind color, but placing in component. Not ideal, but I think this whole Revealer can likely be deprecated at some point or reworked.

Comment (DFurnes) [reply resources/assets/components/Revealer/revealer.scss]: Makes sense! Let's just make sure we don't forget that context!

Comment (weerd) [reply resources/assets/components/Revealer/revealer.scss]: Sounds good! Added the TODO! Twas a good call :)

**Pull request merged**